COPYRIGHT

```diff
@@ -0,0 +1,5 @@
+© Copyright 2015-2022 Adobe. All rights reserved.
```

Comment:
Does this need to be updated to `2015-2023`? It doesn't look like we update the year in other repos, however.